apps/console/src/features/core/components/header.tsx

```diff
@@ -148,6 +150,15 @@ export const Header: FunctionComponent<HeaderPropsInterface> = (
         feature.organizations
     ]);
 
+    useEffect(() => {
+        commonConfig
+            ?.header
+            ?.getUserDropdownLinkExtensions(tenantDomain, associatedTenants)
+            .then((response: HeaderLinkCategoryInterface[]) => {
```

Comment:
```suggestion
                .then((response: HeaderLinkCategoryInterface[]) => {
```

Reply:
There is an eslint warning when indented this way.